assets/js/plugin-check-admin.js

```diff
@@ -247,13 +249,17 @@
 				) {
 					isSuccessMessage = false;
 				}
-				renderResults( results );
+				allResults.push( results );
 			} catch ( e ) {
 				// Ignore for now.
 			}
 		}
 
 		renderResultsMessage( isSuccessMessage );
+
+		for ( let i = 0; i < allResults.length; i++ ) {
+			renderResults( allResults[ i ] );
+		}
```

Comment:
Yah, I noticed that in the bigger plugins. I have reverted that loop and kept as it is. To fix this, I have prepended the final output message instead of appending. If this approach is also acceptable then we can re-explore another approaches. :-)